packages/curve-ui-kit/.storybook/main.ts

```diff
@@ -11,7 +11,11 @@ function getAbsolutePath(packageName: string) {
 }
 
 const config: StorybookConfig = {
-  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|mjs|mts|ts|tsx)'],
+  stories: [
+    '../src/**/*.mdx',
+    '../src/**/*.stories.@(js|jsx|mjs|mts|ts|tsx)',
+    '../../../apps/**/*.stories.@(js|jsx|mjs|mts|ts|tsx)',
```

Comment:
bit weird to import from apps, tbh storybook should be in `/apps` imo
good solution for the time being tho